aliasmgr/aliasmgr_test.go

```diff
@@ -80,7 +88,11 @@ func TestAliasLifecycle(t *testing.T) {
 	require.NoError(t, err)
 	defer db.Close()
 
-	aliasStore, err := NewManager(db)
+	linkUpdater := func(shortID lnwire.ShortChannelID) error {
+		return nil
```

Comment:
nit: Should probably add some test coverage here to see the link updater is actually called when expected.

Reply:
Right. Changed the func to write to a chan, then i'm reading from it when updates are meant to occur